docs/guides/medical.md

```diff
@@ -0,0 +1,181 @@
+	2. **Zbadaj pacjenta** – Możemy sprawdzić ciśnienie krwi oraz tętno. Tylko medyk jest w stanie dokładnie określić wynik pomiaru, tj. w wartościach liczbowych.
+	3. **Bandaże/złamania** – Wyświetla wszystkie bandaże będące na wyposażeniu twoim lub kolegi, które można użyć do opatrzenia rany oraz opcję założenia/ściągnięcia stazy.
+	4. **Leki** – Wyświetla wszystkie leki będące na wyposażeniu twoim lub kolegi. Adrenalinę i morfinę aplikujemy w kończyny, APAP doustnie.
+	5. **Zaawansowane zabiegi** – Głównie przeznaczone dla medyka. Znajdują tu się akcje wykonania RKO tj. Resuscytacja krążeniowo-oddechowa (opcja dostępna dla wszystkich), zaszycie ran oraz przetoczenie krwi, osocza lub soli fizjologicznej.
+	6. **Ciągnij/Nieś** – Możemy przeciągnąć rannego lub go przenieść na plecach. Wybór metody jest uzależniony od danej sytuacji.
+	7. **Przełącz** – Możliwość przełączenia między twoim menu medycznym a kolegi będącym w pobliżu.
+
+## Bandaże, opaski uciskowe
+
+Bandaże mają przypisaną skuteczność dla danej rany, aczkolwiek z racji długiego czasu na otwarcie rany po użyciu najgorszego bandaża, praktycznie nie zwraca się na to uwagi (dotyczy tylko naszej wersji systemu medycznego). Najważniejszym zadaniem jest zatamowanie krwawienia u siebie lub kolegi. Na dobór bandaża warto zwrócić uwagę w momencie kiedy nie mamy dostępu do szycia, wtedy rzadziej będą otwierać się rany. <br/>
+Pamiętaj, o ile nie musisz, nie obciążaj medyka w trudnych sytuacjach. Jeśli jesteś przytomny to najpewniej możesz opatrzyć się sam. Korzystaj z pomocy medyka kiedy wiesz, że ten akurat nie zajmuje się nieprzytomnymi. 
+
+Cechy staz taktycznych:
+
+- zakładane tylko na ręce i nogi,
+- drastycznie szybsze w użyciu od bandażowania (kupują czas medykowi na opatrzenie torsu i głowy, piechocie pozwalają na nieprzerwane prowadzenie ognia w przypadku obrażeń kończyn),
+- wielokrotnego użytku - po ściągnięciu wracają do ekwipunku ściągającego,
+- po kilku minutach od założenia powodują narastający ból,
+- wlewy medyczne (krew, morfina) nie przynoszą efektu przy podaniu drogą zastazowanej kończyny do czasu jej zdjęcia.
+
+`++` = Super | `+` = Zwiększona | Brak = Standardowa
+
+| Rana\Bandaż	| Jałowy			|Elastyczny	|Uciskowy	| QuikClot		|  Pol\Eng |
+|:-------------:|:-----------------:|:---------:|:---------:|:-------------:|:--------:|
+| Draśnięcie	| ++				| 			| 			| 				| Scrape |
+| Płatowa		|					| ++ 		| 			| 				| Avulsion |
+| Stłuczenie	|	n/d				| n/d 		| n/d		| n/d			| Bruise |
+| Zgniecenie	| 					| ++ 		| 			| 				| Crushed |
+| Cięta			| ++ 				| 			| 			| 				| Cut |
+| Rozerwana		| ++ 				| 			| 			| 				| Tear |
+| Postrzałowa	| + 				| + 		| ++ 		| ++ 			| Velocity |
+| Kłuta			| + 				| + 		| ++ 		| ++ 			| Puncture |
+| **Pol\Eng**	|**Field Dressing**	|**Elastic**|**Packing**|**QuikClot**	|**Dressing\Wound**|
```

Comment:
This `Pol\Eng` is a bit misleading.

You can change that to `Pol/Eng` as that would fit a bit better or leave empty cells and give a quick explanation just beneath table legend.